src/pages/groupComparison/GroupComparisonUtils.tsx

```diff
@@ -335,8 +333,55 @@ export function getOverlapFilteredGroups(
     });
 }
 
-export function ENRICHMENTS_NOT_2_GROUPS_MSG(tooMany:boolean) {
-    return `We can only show enrichments when two groups are selected. Please ${tooMany ? "deselect" : "select"} groups in the 'Active Groups' section so that only two are selected.`;
+export function MakeEnrichmentsTabUI(
+    getStore:()=>GroupComparisonStore,
+    getEnrichmentsUI:()=>MobxViewAlwaysComponent
+) {
+    return MakeMobxView({
+        await:()=>{
+            const store = getStore();
+            const ret:any[] = [store._selectedGroupsNotOverlapRemoved, store.activeGroups, store.activeStudyIds];
+            if ((store.activeGroups.isComplete &&
```

Comment:
may be logical not instead of empty if block

Reply:
yeah I wasn't sure about it but I think when its written this way it's easier to understand the logic. no?